src/ol_superset/pythonpath/superset_config.py

```diff
@@ -35,7 +35,7 @@
 # --------------------------------
 APP_NAME = "MIT OL Business Intelligence"
 # Specify the App icon. Useful for white-labeling
-APP_ICON = "/static/assets/images/superset-logo-horiz.png"
+APP_ICON = "../static/assets/images/OL_sub-brand_lockup_one-line_rgb_black.png"
```

Comment:
I think this gets rendered as the path in the web UI for loading the image, so it shouldn't have the leading `../`. You'll also need to add a `COPY` command in the Dockerfile to place the logo into the final image at the proper location.